en/faq/now-deployment.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: How to deploy with Now?
-description: How to deploy Nuxt.js app with Now?
+title: How to deploy with Vercel?
+description: How to deploy Nuxt.js app with Vercel?
 ---
 
 ![nuxt-now-builder](https://user-images.githubusercontent.com/904724/61308402-7a752d00-a7f0-11e9-9502-23731ccd00fd.png)
 
-## Now V2
+## Vercel
 
-To deploy with [Now V2](https://zeit.co/now), the Nuxt.js team and contributors worked on an official [@nuxtjs/now-builder](https://github.com/nuxt/now-builder) package.
+To deploy with [Vercel](https://vercel.com), the Nuxt.js team and contributors worked on an official [@nuxtjs/now-builder](https://github.com/nuxt/now-builder) package ('Now' is Vercel previous name).
 
 All you have to do is to setup a `now.json` file:
```

Comment:
Shouldn't it also be a vercel.json file instead of now.json?

Reply:
In theory: yes – but In practise this is related to [nuxt/now-builder](https://github.com/nuxt/now-builder), so _(I believe)_ would be necessary some refactoring on that package. Not related to this documentation.

Reply:
great, thanks for the clarification